src/notation/internal/notationstyle.cpp

```diff
@@ -89,3 +89,53 @@ Notification NotationStyle::styleChanged() const
 {
     return m_styleChanged;
 }
+
+mu::io::path NotationStyle::selectStyleFile(bool forLoad)
```

Comment:
This is not a good place for this code.
We adhere to the MVVM architectural pattern
https://en.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93viewmodel
User interaction must be at the ViewModel level
This place is Model, there should be no user interaction, ideally. Otherwise, this code (loading, saving styles) cannot be used in other cases, for example, during auto testing, or on the backend, or anywhere else.